<?xml version="1.0"?>
<launch>

  <!-- Synchronize depth images and camera info at reception side -->
  
	<node name="vva_image_camera_info_sync" pkg="vva_topics_sync" type	="vva_image_camera_info_sync.py" output="screen">
      <param name="input_image_topic"        type="string" value="/camera/depth_registered/image_raw"/>
      <param name="input_camera_info_topic"  type="string" value="/camera/depth_registered/camera_info"/>
      
      <param name="output_image_topic"       type="string" value="/synchronized/camera/depth_registered/image_raw"/>
      <param name="output_camera_info_topic" type="string" value="/synchronized/camera/depth_registered/camera_info"/>
	</node>
  
</launch>


